packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -2,36 +2,332 @@
 # Schema
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
-# Category
-registration:
-  # Name
-  view:
-    # Metric Parameters
-    type: event
+event:
+  name:
+    type: string
     description: |
-      Registration form displayed
-    metadata:
-      tags:
-        - registration
+      The name of the event
+    lifetime: ping
+    send_in_pings:
+      - accounts-events
     notification_emails:
       - fxa-staff@mozilla.com
     bugs:
-      - https://bugzilla.mozilla.org/123456789/
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
     data_reviews:
-      - http://example.com/path/to/data-review
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
     expires: never
+    data_sensitivity:
+      - web_activity
 
-  submit:
-    type: event
+account:
+  uid:
+    type: string
     description: |
-      Registration form submission
-    metadata:
-      tags:
-        - registration
+      The account's uid
+    lifetime: ping
+    send_in_pings:
+      - accounts-events
     notification_emails:
       - fxa-staff@mozilla.com
     bugs:
-      - https://bugzilla.mozilla.org/123456789/
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
     data_reviews:
-      - http://example.com/path/to/data-review
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
     expires: never
+    data_sensitivity:
+      - highly_sensitive
+
+relying_party:
+  service:
+    type: string
+    description: |
+      The service name of the relying party
+    lifetime: ping
+    send_in_pings:
+      - accounts-events
+    notification_emails:
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+    expires: never
+    data_sensitivity:
+      - technical
+
+  oauth_client_id:
+    type: string
+    description: |
+      The client id of the relying party
+    lifetime: ping
+    send_in_pings:
+      - accounts-events
+    notification_emails:
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+    expires: never
+    data_sensitivity:
+      - technical
+
+session:
+  flow_id:
+    type: string
+    description: an ID generated by FxA for its flow metrics
+    lifetime: ping
+    send_in_pings:
+      - accounts-events
+    notification_emails:
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+    expires: never
+    data_sensitivity:
+      - web_activity
+
+  country:
```

Comment:
As mentioned before our pipeline automatically adds metadata based on GeoIP. What's the reason this is tracked separately?
It might be ok to do that, but I would then also recommend noting that in the description, so people looking in the data later won't get confused by two country values in the dataset.